DMTN-266.tex

```diff
@@ -48,6 +49,178 @@
+
+To construct a \texttt{lsst.afw.cameraGeom.Camera} object, \texttt{gbdesAstrometricFitTask} should be run with the configuration option \texttt{saveCameraObject=True}.
+
+\section{Outstanding issues}\label{sec:out}
+There are a few known issues that we plan to work on, but which are either in early stages of development or are still in the planning stage:
+\begin{itemize}
+\item \textbf{Modeling Atmospheric Turbulence} While exposure times for the HSC precursor data are generally around $300$ seconds, the exposure times for LSST will be either $15$ or, more likely, $30$ seconds. Because of this, atmospheric turbulence is expected to have much stronger effects. \cite{Fortino2021} and \cite{Leget2021} showed that it is possible to model astrometric residuals due to atmospheric turbulence using Gaussian Processes. Initial tests showed that a Gaussian Proceses model was able to capture a large portion of the structured astrometric residuals in simulated ComCam data. Figures~\ref{fig:gp vec} through~\ref{fig:gp resid} show the results of the Gaussian Processes fit for a sample exposure.
+\begin{figure}
+\includegraphics[width=0.7\columnwidth]{figures/vec_test_dv.png}
+\caption{Sample astrometric residuals for a simulated ComCam exposure, fit with a low-order polynomial in order to not fit atmospheric turbulence.}
+\label{fig:gp vec}
+\end{figure}
+\begin{figure}
+\includegraphics[width=0.7\columnwidth]{figures/predict_test_dv.png}
+\caption{Gaussian Processes prediction for the residuals in Figure~\ref{fig:gp vec}.}
+\label{fig:gp predict}
+\end{figure}
+\begin{figure}
+\includegraphics[width=0.7\columnwidth]{figures/resid_test_dv.png}
+\caption{Difference between the astrometric residuals and their prediction using Gaussian Processes.}
+\label{fig:gp resid}
+\end{figure}
+However, more work is necessary to add this to the pipeline. We will need to confirm that it works well in a larger range of observing conditions on real data, and to ensure that it is not prohibitively expensive in terms of computing resources.
+\item \textbf{Evolution in the Camera} We know that cameras evolve over time, corresponding to various factors such as when the instrument is warmed up and cooled down again. These changes in the instrument lead to changes in the astrometric distortion, which are visible as discontinuities in the distribution of the best-fit visit model parameters. The most straightforward approach to dealing with this will be to run \texttt{gbdesAstrometricFit} separately for each period of relative stability. A more sophisticated approach might be to allow per-epoch camera models that are fit simultaneously, but it is not clear whether the potential improvement in precision will be worth the added complexity in the fit. Implementing the former solution will be primarily a question of bookkeeping, but will probably require that the pipeline middleware allows for versioned \texttt{Camera} objects, which still need to be developed.
```

Comment:
I'm not sure why we need middleware versioned Camera objects.  I thought the plan was that the camera distortion model could itself be a calibration product with associated timespans and this could be handed to the Prompt processing task; or could be used by gbdes.
Also, isn't there the other opportunity to fit multiple epochs simultaneously, and fixing the camera model for each of those epochs (e.g. multi-epoch Figure 6)?  (Though I might be missing something in why that wouldn't scale well).

Reply:
Maybe a versioned camera object is not the right way to describe this, though I thought that was how @TallJimbo has referred to camera objects that correspond to different time periods. The overarching issue, though, is that there needs to be a middleware dimension corresponding to epochs, which doesn't currently exist.

For the second comment, this is what I am trying to describe in the sentence "A more sophisticated approach..."

Reply:
Ok, thanks for the clarification @TallJimbo. In any case, I'm going to just take out the last sentence in this paragraph.